pkg/api/types.go

```diff
@@ -2213,6 +2213,17 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will will be default or zero if there is
```

Comment:
Got it, then how about update the sentence `default or zero if there is no default` a bit?

Such as when using default one and when setting to zero? Using `default or zero if there is no default` may make people confused, as they may not clear when using default and when using zero?